Source/WebKitLegacy/mac/DOM/DOMText.mm

```diff
@@ -53,7 +53,9 @@ - (DOMText *)splitText:(unsigned)offset
 - (DOMText *)replaceWholeText:(NSString *)content
 {
     WebCore::JSMainThreadNullState state;
-    return kit(IMPL->replaceWholeText(content).get());
+    RefPtr protectedImpl = IMPL;
+    protectedImpl->replaceWholeText(content);
```

Comment:
Why not just: `Ref { IMPL }->replaceWholeText(content)`?

Reply:
I hesitated. I switched to your proposal.